include/envoy/config/grpc_mux.h

```diff
@@ -47,6 +47,20 @@ class GrpcMuxWatch {
 
 typedef std::unique_ptr<GrpcMuxWatch> GrpcMuxWatchPtr;
 
+/**
+ * Handle critical gRPC log messages.
+ */
+class GrpcMuxLogger {
+public:
+  virtual ~GrpcMuxLogger() {}
```

Comment:
`= default` is more C++11 style than empty destructor.

Reply:
I thought it was Envoy's particular code style since other interfaces have an empty destructor as well.

Reply:
Could be. Not a regular contributor here myself, but seemed weird to me.

Reply:
Yes, this is Envoy style.

Reply:
Out of curiosity, why?

Reply:
@mattklein123 can probably comment on this one, it predates my involvement in the project.

Reply:
Habit, no reason. It doesn't matter one way or the other and consistency is more important. If someone would like to go and fix all cases in the repo we can switch.